crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -881,7 +874,7 @@ def delete_upgrade_scripts(node)
           "post-upgrade",
           "chef-upgraded"
         ].map { |f| "/usr/sbin/crowbar-#{f}.sh" }.join(" ")
-        scripts_to_delete << "/etc/neutron/lbaas-connection.conf"
+        scripts_to_delete << " /etc/neutron/lbaas-connection.conf"
```

Comment:
maybe add some comment why this conf file is deleted together with scripts?

Reply:
Good point, I have no idea. (That was not the point of the fix however)

Reply:
I think that file is specific to the lbaas migration tool. It contains the connection details to the neutron database.  But you're right that more something for a separate commit.